zingolib/Cargo.toml

```diff
@@ -52,20 +52,23 @@ group = "0.13"
 rust-embed = { version = "6.3.0", features = ["debug-embed"] }
 orchard = { workspace = true }
 subtle = "2.4.1"
-incrementalmerkletree = { workspace = true }
+incrementalmerkletree = { workspace = true, features = ["test-dependencies"] }
 zcash_address = { workspace = true }
 zcash_client_backend = { workspace = true }
 zcash_encoding = { workspace = true }
 zcash_note_encryption = { workspace = true }
 zcash_primitives = { workspace = true }
 zcash_proofs = { workspace = true }
+zcash_client_sqlite = {workspace = true }
```

Comment:
Do we still need this?

Reply:
Turns out we do! We're using their helpers for `read_shard` and `write_shard`

Reply:
Should we request that those helpers move?